ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/core/RemoteStartTransactionRequest.java

```diff
@@ -83,19 +84,19 @@ public void setConnectorId(Integer connectorId) throws PropertyConstraintExcepti
     /**
      * The identifier that Charge Point must use to start a transaction.
      *
-     * @return an {@link IdToken}.
+     * @return an IdToken.
      */
-    public IdToken getIdTag() {
+    public String getIdTag() {
         return idTag;
     }
 
     /**
      * Required. The identifier that Charge Point must use to start a transaction.
      *
-     * @param idTag    an {@link IdToken}.
+     * @param idTag an IdToken
      */
     @XmlElement
-    public void setIdTag(IdToken idTag) {
+    public void setIdTag(String idTag) {
```

Comment:
This method should validate the input and throw an exception if it's invalid. Fail fast. - This should be verified by test.